tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -528,7 +528,7 @@ impl<'writer> Writer<'writer> {
         self.is_ansi
     }
 
-    pub(in crate::fmt::format) fn bold(&self) -> Style {
+    pub(in crate::fmt::format) fn bold_style(&self) -> Style {
```

Comment:
I renamed them, because I screwed up a few times, and was worried that it might happen to people who touch the code in the future.

I imported `owo_colors::OwoColorize`, and got a `.bold()` method that was [implemented for every single object](https://github.com/jam1garner/owo-colors/blob/0f32923c5cef107d46221dd53bcee5ce8868d3a0/src/lib.rs#L496-L497). Which occasionally led to me calling the wrong method. The final solution doesn't import `owo_colors::OwoColorize` in this file anymore though.